modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldFeedback.tsx

```diff
@@ -19,13 +19,17 @@ export default function FieldFeedback({
 
 	return (
 		<ClayForm.FeedbackGroup className="field-feedback" {...otherProps}>
+			{helpMessage && (
+				<ClayForm.FeedbackGroup>
```

Comment:
We don't need this here since it's declared right above. `<ClayForm.FeedbackGroup>` is the container for `HelpText` and `FeedbackItem`.